libs/test-common/src/main/java/org/codice/ddf/test/common/annotations/PaxExamRule.java

```diff
@@ -128,8 +141,6 @@ private void failRule(String testClassName, Throwable throwable, String failureM
   }
 
   private void finished(Description description) {
-    LOGGER.info("Finished {} ({}/{})", description.getMethodName(), testsExecuted, testCount);
```

Comment:
❓ Why was this removed instead of being moved to debug or trace?

Reply:
Because of the mostly redundant logs that are now produced by the `ExamResultLogger`:
https://github.com/codice/ddf/pull/5396/files#diff-d79256616b2290cfe3b63bfe7877fcabR25-R32